sw/device/lib/testing/test_main.c

```diff
@@ -53,5 +53,5 @@ int main(int argc, char **argv) {
   test_status_set(result ? kTestStatusPassed : kTestStatusFailed);
 
   // Unreachable code.
-  return 1;
+  return kTestStatusFailed;
```

Comment:
Just return `1` here, as this is unreachable anyway

Reply:
Done.